### PR TITLE
update hook position for performance measurement

### PR DESCRIPTION
**Pull request by ttyio**
use before_run() instead of begin() as start time of each inference iteration.

### CODE REVIEW

File: tftrt/examples/image-classification/image_classification.py [hunk 1]
```diff
@@ -38,13 +38,12 @@ def __init__(self, batch_size, num_records, display_every):
         self.num_steps = (num_records + batch_size - 1) / batch_size
         self.batch_size = batch_size
 
-    def begin(self):
+    def before_run(self, run_context):
```
Comment (pooyadavoodi) [inline tftrt/examples/image-classification/image_classification.py]: @ttyio I think this is already changed in the master branch to `before_run`.

File: tftrt/examples/image-classification/image_classification.py [hunk 2]
```diff
@@ -38,13 +38,12 @@ def __init__(self, batch_size, num_records, display_every):
...
         self.start_time = time.time()
 
     def after_run(self, run_context, run_values):
         current_time = time.time()
         duration = current_time - self.start_time
-        self.start_time = current_time
```
Comment (pooyadavoodi) [inline tftrt/examples/image-classification/image_classification.py]: But this change is a good change.

**Comment by pooyadavoodi** [comment]
@ttyio I think a rebase onto master branch will prepare this PR for a merge.

**Comment by ttyio** [comment]
Thanks @pooyadavoodi , I have rebase the branch, and the PR is updated.

**Comment by pooyadavoodi** [comment]
Thanks @ttyio for the PR.

**Pull request merged**